app/models/message.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class Message < ApplicationRecord
+  acts_as_paranoid
+  belongs_to :user
```

Comment:
The same situation here:
```suggestion
  belongs_to :user
  belongs_to :conversation
```